### DISCUSSION TITLE
엔터프라이즈 애플리케이션 아키텍처 패턴 참여 못하는 분들에 대한 기록

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#398] [General]
현 시점에서 세분이 참여가 어렵거나 못하겠다는 얘기를 해 주셔서 진행을 안하고
다음 책인 < 객체지향의 사실과 오해 > 부터 진행하는 걸로 얘기가 되었습니다.

정리해 보면

- 이정안님
  - 엔터프라이즈 애플리케이션 아키텍처 패턴 첫 모임 4일 전인 3월 4일 discussions를 통해 전달
  - https://github.com/ThinkAboutSoftware/AcademicConference/discussions/387
- 이예신님
  - 정안님과 같은 날 역시 discussions를 통해 전달
  - https://github.com/ThinkAboutSoftware/AcademicConference/discussions/388
- 윤지윤님
  - 3월 20일, 책 난이도와 부트캠프 프로젝트 병행이 어려워 개인 카톡을 통해 전달
- 홍성현님
  - 4월 5일, 세 번째 책 모임 날 카톡으로 취업 준비로 준비가 미흡해 다음 책 부터 준비하겠다고 알림

**Comment by fkdl0048** [comment]
현재 날짜 기준으로 다음주에 진행되는 건가요?

**Reply by jongfeel** [reply]
네 맞습니다.
다음 주가 엔터프라이즈 애플리케이션 아키텍처 패턴 세 번째 모임입니다.

**Reply by fkdl0048** [reply]
제가 잘못 이해했네요, 이탈자가 많아서 바로 객체지향의 사실과 오해책 부터 진행하는 줄 알았습니다..

**Reply by jongfeel** [reply]
아항 설명을 좀 더 드리자면...

Academic conference가 4년차가 되다 보니 과거 기록을 잘 적어 두는 게 좋겠다는 생각이 들었습니다.
매년 멤버들의 참여 의사를 discussions에 적으시는 분도 있지만 그렇지 않은 분도 있어서요.
또 올해 안으로 README.md 파일도 업데이트도 해보면 좋을 것 같다는 생각에 **기록용** 으로 적은 거라 보시면 됩니다.
몇 년 뒤에 이 글을 보면 그 때 이해가 되실 거에요.

**Comment by jongfeel** [comment]
4월 5일 홍성현님도 이 책은 건너뛴다고 알려서 본문 내용 추가 업데이트 함
